build.sbt

```diff
@@ -7,7 +7,7 @@ enablePlugins(TutPlugin)
 
 lazy val buildSettings = Seq(
   organization := "io.github.finagle",
-  version := "0.3.1",
+  version := "0.3.2-snapshot",
```

Comment:
Can you just change this to `0.3.2`? Then we'll make a new release once it's merged.

Otherwise LGTM.